lib/serverengine/server.rb

```diff
@@ -66,12 +66,14 @@ def install_signal_handlers
       s = self
       SignalThread.new do |st|
         st.trap(Daemon::Signals::GRACEFUL_STOP) { s.stop(true) }
-        st.trap(Daemon::Signals::IMMEDIATE_STOP) { s.stop(false) }
-        st.trap(Daemon::Signals::GRACEFUL_RESTART) { s.restart(true) }
-        st.trap(Daemon::Signals::IMMEDIATE_RESTART) { s.restart(false) }
-        st.trap(Daemon::Signals::RELOAD) { s.reload }
         st.trap(Daemon::Signals::DETACH) { s.stop(true) }
-        st.trap(Daemon::Signals::DUMP) { Sigdump.dump }
+        unless $platformwin
```

Comment:
How should we stop ServerEngine on Windows? I'm asking this because README.md will need additional notes for Windows platform.